snippets/csharp/concepts/linq/how-to-query-a-collection-of-objects_1.cs

```diff
@@ -78,7 +74,7 @@ public class Program
     {
         public static void Main()
         {
-            StudentClass sc = new StudentClass();
+            var sc = new Student();
```

Comment:
Consistent with the change I requested above, the `sc` object isn't needed, and the next line would be `Student.QueryHighScores(1,90);`